__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    'name': 'Purchase Order Enhancement',
+    'version': '1.0',
+    'license': 'LGPL-3',
+    'description': 'This module contains enhanced features for Purchase module',
+    'depends': ['base', 'purchase'],
+    'author': 'John Bui',
+    'application': True,
```

Comment:
This module contains customizations for the **_purchase_** module of Odoo. This module itself is not an application. So please mark _**'application'**_ as **_False_** here